src/test/java/uk/gov/hmcts/reform/sendletter/controllers/sendlettercontroller/SendLetterWithPdfsAndCopiesControllerTest.java

```diff
@@ -77,6 +78,14 @@ void should_return_403_if_service_throws_ServiceNotConfiguredException() throws
             .andExpect(status().isForbidden());
     }
 
+    @Test
+    void should_return_401_if_service_throws_UnauthenticatedException() throws Exception {
+        given(authService.authenticate(anyString())).willThrow(new UnauthenticatedException("Hello"));
+
+        sendLetter(Resources.toString(getResource("controller/letter/v3/valid_letter.json"), UTF_8))
+            .andExpect(status().isUnauthorized());
+    }
```

Comment:
can you add 403 response in end point swagger as ApiResponse ?

Reply:
https://github.com/hmcts/send-letter-service/pull/593